tests/gocase/unit/introspection/introspection_test.go

```diff
@@ -45,6 +45,11 @@ func TestIntrospection(t *testing.T) {
 		require.Regexp(t, "id=.* addr=.*:.* fd=.* name=.* age=.* idle=.* flags=N namespace=.* qbuf=.* .*obuf=.* cmd=client.*", v)
 	})
 
+	t.Run("CLIENT INFO", func(t *testing.T) {
+		v := rdb.Do(ctx, "CLIENT", "INFO").Val()
```

Comment:
note go-redis does not support CLIENT INFO yet (a pending PR, should be supported soon)
so here i am using Do